src/operations/update.ts

```diff
@@ -32,9 +52,21 @@ export class UpdateOperation implements Operation<UpdateOperationInput> {
 
   public async execute(input: UpdateOperationInput) {
     const ddb = createClient(input.region);
+    const item = input.expressionAttributeValues || await this.read(input.expressionAttributeFiles!);
+    
     await ddb.update({
       TableName: input.table,
       Key: input.key,
+      UpdateExpression: `set ${input.updateExpression} = :${input.updateExpression}`,
+      ExpressionAttributeValues: {
+        [`:${input.updateExpression}`]:`${item}`
+      }
     }).promise();
   }
+
+  private async read(path: string) {
+    const content = await fs.readFile(path, { encoding: "utf8" });
+
+    return JSON.parse(content);
```

Comment:
there is no reason to expect the content to  be json but it works so good enough